package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "geoasteroids",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node app.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jsolly/GeoAsteroids.git"
+  },
+  "author": "jsolly",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jsolly/GeoAsteroids/issues"
+  },
+  "homepage": "https://github.com/jsolly/GeoAsteroids#readme",
+  "dependencies": {
+    "express": "^4.18.1",
+    "serve-favicon": "^2.5.0"
```

Comment:
It's usually preferable decreasing number of dependencies. Considering static serving files, you can place a favicon into `/public`.